src/Raven.Server/ServerWide/ClusterStateMachine.License.cs

```diff
@@ -203,6 +208,8 @@ private void AssertLicense(ClusterOperationContext context, string type, Blittab
             AssertRefreshFrequency(databaseRecord, newLicenseLimits, context);
             AssertSorters(databaseRecord, newLicenseLimits, context, items, type);
             AssertAnalyzers(databaseRecord, newLicenseLimits, context, items, type);
+            AssertSnowflakeEtl(databaseRecord, newLicenseLimits);
```

Comment:
why we have snowflake here, but I do not see other ETLs in those assertions? Is this correct?